redux/main/thunks.ts

```
@@ -79,23 +79,114 @@ export const uploadIncome = createAsyncThunk(
         headers: {
           'content-type': 'application/json',
           Authorization: `Bearer ${token}`,
         },
-        body: JSON.stringify(rest),
+        body: JSON.stringify({name, color: color.split('#')[1], groupId}),
+      });
+      data = await resp.json();
+      if (data.err) throw data.err;
+      thunkAPI.dispatch(fetchIni());
+    } catch (err) {
+      throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)